### PR TITLE
ps2: week 7-8: build user interface with basic api integration

### PR DESCRIPTION
**Pull request by Pulkit007**
## Issue that this pull request solves

Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [ ] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x ] I have performed a self-review of my own code
-   [x ] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ x] My changes generate no new warnings
-   [ x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by rakshit512002** [comment]
Looks Good To Me @akshatnema

**Comment by akshatnema** [comment]
Due to change in models.py, waiting for review from @PranshuNayak and @samay-rgb

File: FusionIIIT/applications/ps2/models.py
Comment (samay-rgb) [inline FusionIIIT/applications/ps2/models.py]: You can use ```quantity``` instead of ```no_of_quantity```

**Review by PranshuNayak — APPROVED** [review]

**Review by PranshuNayak — COMMENTED** [review]
Push the migration files for this model.

**Comment by akshatnema** [comment]
@PranshuNayak @samay-rgb kindly review the PR again

**Review by PranshuNayak — COMMENTED** [review]
Please push only a single migration file for this model

**Comment by Pulkit007** [comment]
> Please push only a single migration file for this model

@PranshuNayak Done.

**Review by PranshuNayak — APPROVED** [review]
LGTM!

**Review by samay-rgb — APPROVED** [review]

**Pull request merged**
